docs/guides/Integration-Guides/tax-service-integration-guide/tax-services-specification.md

```diff
@@ -142,57 +185,87 @@ In response to the request sent by Checkout, we expect an array of products, eac
-| `id` | string | Request item index, which means the SKU's position on the `items` array sent by the request body. |
-| `taxes` | array | List of all the taxes types for an SKU. |
-| `name` | string | Tax name that will appear on the checkout. |
-| `description` | string | Informative field, which does not appear on the storefront. |
-| `value` | number | Absolute numeric value that will be added to the original price. |
+| Field         | Type   | Description                                                                                       |
+| ------------- | ------ | ------------------------------------------------------------------------------------------------- |
+| `id`          | string | Request item index, which means the SKU's position on the `items` array sent by the request body. |
+| `taxes`       | array  | List of all the taxes types for an SKU.                                                           |
+| `name`        | string | Tax name that will appear on the checkout.                                                        |
+| `description` | string | Informative field, which does not appear on the storefront.                                       |
+| `value`       | number | Absolute numeric value that will be added to the original price.                                  |
 
 In the example above, the only item in the items array has a cost of `10`, and, including the calculated taxes returned by the tax calculation tool, the total value would be `10 + 3.48 + 22 = 35.48`.
 
 >ℹ️ If no taxes apply to the items in the order, the expected response is an empty array (`[]`).
 
->ℹ️ For the Checkout API to understand the request body, the content-type must be set to `application/vnd.vtex.checkout.minicart.v1+json`.
+For the Checkout API to understand the request body, the `content-type` must be set to `application/vnd.vtex.checkout.minicart.v1+json`.
 
 #### Jurisdiction fields
 
-If you use Avalara as your tax calculation provider, response bodies might also include the following fields, which refer to the different jurisdictions that may apply according to location.
+If you use [Avalara](https://www.avalara.com/us/en/index.html) as your tax calculation provider, response bodies might also include the following fields, which refer to the different jurisdictions that may apply according to location.
 
-| Field | Type | Description |
-| - | - | - |
-| `jurisType` | string | Type of jurisdiction that applies to calculation. |
+| Field       | Type   | Description                                               |
+| ----------- | ------ | --------------------------------------------------------- |
+| `jurisType` | string | Type of jurisdiction that applies to calculation.         |
 | `jurisCode` | string | Unique code that identifies the appropriate jurisdiction. |
 | `jurisName` | string | Name of the jurisdiction that applies to the calculation. |
 
 These fields are also read by Checkout and added to the `priceTag`.
```

Comment:
What is the `priceTag`? Is it a field in the orderForm? Worth linking to more explanation